core/railing/railing.py

```diff
@@ -38,14 +39,9 @@ def make_fill(bm, face, prop):
     bmesh.ops.translate(bm, verts=top_edge.verts, vec=Vector((0., 0., -1.))*prop.corner_post_width/2)
 
     # create railing top
-    ret = bmesh.ops.duplicate(bm, geom=[top_edge])
-    top_dup_edge = filter_geom(ret["geom"], BMEdge)[0]
-    horizon = edge_vector(top_dup_edge).cross(Vector((0., 0., 1.)))
-    up = edge_vector(top_dup_edge)
-    up.rotate(Quaternion(horizon, math.pi/2).to_euler())
-    edge_to_cylinder(bm, top_dup_edge, prop.corner_post_width/2, up)
-    bmesh.ops.translate(bm, verts=top_edge.verts, vec=Vector((0., 0., -1.))*prop.corner_post_width/2)
+    create_railing_top(bm, top_edge, prop)
```

Comment:
Oops, Just noticed that this was not part of the `make_corner_posts` function. It seemed like it was in the diff.

In that case then the skip is not necessary in line 25. However, at this point